src/core/bau_inputs.jl

```diff
@@ -45,6 +45,7 @@ function BAUInputs(p::REoptInputs)
     techs = Techs(p, bau_scenario)
 
     boiler_efficiency = Dict{String, Float64}()
+    fuel_cost_per_kwh = Dict{String, AbstractArray}()
```

Comment:
Yes, good idea on the CHP-basis. All costs (capital, O&M, and fuel) are based on electric capacity or production. That's pretty industry-standard/implied, but I agree with making sure that's stated in the docs.

Reply:
I guess I didn't actually avoid units conversion within the JuMP model because the fuel burn slope/intercept for Generator are still "per gallon" - but the `dvFuelUsage` is always now in kWh, and the the `FuelCosts` for fuel burning techs are always just `dvFuelUsage* p.fuel_cost_per_kwh`.

Reply:
Got it, thank you for clarifying - I misunderstood what the units were in the first place.  If we're going to ask for inputs in MMBTU and run the optimization in kWh, then the conversion has to happen somewhere regardless.  I think the broader commentary is that kWh-thermal and kWh-electric might need to be called out for technologies that are involved with both heating and electric loads, if we are using kWh of fuel energy (i.e., heat, but not for heating load necessarily), we should document that.  

I don't think we need to change anything, and I'll review the math documentation to check the units in the formulation, but the units reconcile in our code implementation from what I'm seeing.   Just for ease of use and development, it might be good to put units up on the Wiki where we have the inputs mapping as well.